pysixdesk/lib/utils.py

```diff
@@ -43,23 +41,19 @@ def download_output(filenames, dest, zp=True):
     '''Download the requested files to the given destinaion.
     If zp is true, then zip the files before download.
     '''
-    status = False
     if not os.path.isdir(dest):
         os.makedirs(dest, 0o755)
 
     for filename in filenames:
+        if not os.path.isfile(filename):
+            raise FileNotFoundError("The file %s doesn't exist, download failed!" % filename)
```

Comment:
The same, `check` is called in `preprocess.py` and `sixtrack.py`, avoid to use `raise` here.